packages/server/lib/browsers/types.ts

```diff
@@ -4,7 +4,7 @@ import type { Automation } from '../automation'
 import type { CDPSocketServer } from '@packages/socket/lib/cdp-socket'
 
 export type Browser = FoundBrowser & {
-  majorVersion: number
+  majorVersion: number | string
```

Comment:
Does this have any impact on the public API? https://docs.cypress.io/api/cypress-api/browser#Syntax

Reply:
it doesn't but independently I think the type is wrong since its a string in both chrome and firefox. I can update it on the docs PR

Reply:
the types are actually correct here https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts#L107 but incorrect in the docs, so I added them to the docs PR https://github.com/cypress-io/cypress-documentation/pull/6103